.github/workflows/pr.yaml

```diff
@@ -300,11 +300,10 @@ jobs:
     - name: Planemo test tools
       run: |
         if grep -lqf .tt_biocontainer_skip changed_tools_chunk.list changed_repositories_chunk.list; then
-                PLANEMO_OPTIONS="--job_config_file job_conf.xml_condor"
+                PLANEMO_OPTIONS="--job_config_file job_conf_no_biocontainers.xml_condor"
```

Comment:
Clever! However, I guess I need to find out why this container is failing.

Reply:
This change still seems to be needed, right? Otherwise the biocontainers skiplist is doing nothing